auth/.now/project.json

```diff
@@ -0,0 +1 @@
+{"projectId":"QmPJxRoHnLpzMgfwYrmzkT6RXfq4HQBQB7Pwn8GvvxoPDX","orgId":"jMaSryOhTf3GpqSJRnP9bcf5"}
```

Comment:
pls add `.now` to `.gitignore`

Reply:
Since it's now in the gitignore, I think you'll probably need to do some fancy things to delete it from the commit